src/main/java/de/uni_tuebingen/ub/nppm/util/Utils.java

```diff
@@ -412,4 +417,28 @@ public static String formatDate(Date date) {
         return date != null ? DATE_FORMAT.format(date) : null;
     }
 
+    //Als Fallback wird die aktuelle Systemzeit zurückgegeben um sicherzustellen das die css datei neu geladen wird
+    public static long getLastModifiedTimestampForCSS(ServletContext context, String path) throws MalformedURLException, URISyntaxException {
+        URL resource = context.getResource(path);
+
+        // Frühzeitiger Ausstieg: kein Zugriff auf Ressource oder kein File-URL
+        if (resource == null || !"file".equals(resource.getProtocol())) {
+            return System.currentTimeMillis(); // Fallback: Cache-Busting sicherstellen
+        }
+
+        File file = new File(resource.toURI());
+
+        // Datei existiert nicht: trotzdem Fallback geben
+        if (!file.exists()) {
+            return System.currentTimeMillis();
```

Comment:
This is a very tolerant solution here, but i wonder if it would be better to throw an exception, else we will never see that something is wrong when testing. Or should we really be tolerant here?

Reply:
Maybe this is not necessary => this is a server sided error that only developers are able to solve and there is something fundamentally wrong (compared to other messages where the user should know that e.g. he requested a non-existing ID)